packages/cli/lib/lib/webpack/push-manifest.js

```diff
@@ -1,9 +1,12 @@
 const createLoadManifest = require('./create-load-manifest');
 
 module.exports = class PushManifestPlugin {
+  constructor(env = {}) {
```

Comment:
Spacing seems off here (or it's my phone). Also the underscore likely trips our lint rules.

Reply:
done.